src/App.scss

```diff
@@ -21,3 +21,32 @@
 .message-body {
   white-space: pre-line;
 }
+
+@media print, screen and (min-width: 769px) {
```

Comment:
The media query `@media print, screen and (min-width: 769px)` might be overly broad. Consider separating print and screen media queries for more precise styling control.